pkg-manager/plugin-commands-installation/src/dedupe.ts

```diff
@@ -3,11 +3,17 @@ import { UNIVERSAL_OPTIONS } from '@pnpm/common-cli-options-help'
 import renderHelp from 'render-help'
 import { type InstallCommandOptions } from './install'
 import { installDeps } from './installDeps'
+import { dedupeDiffCheck } from './dedupeDiffCheck'
 
-export const rcOptionsTypes = cliOptionsTypes
+export function rcOptionsTypes () {
```

Comment:
In a future change, we'll probably want to extend more `install` options as requested over in https://github.com/pnpm/pnpm/issues/6105#issuecomment-1462099984.